schemas/component/definitions.schema

```diff
@@ -33,6 +33,7 @@
             }
         },
         "expression": {
+            "$role": "expression",
             "type": "string",
```

Comment:
This won't help.  The file is not actually used and all of the definitions are in component.schema which is where this change should go.  Gabo has a PR to move this.

Reply:
Updated the new V1 definitions checked in by @gabog